azurerm/resource_arm_redis_cache.go

```diff
@@ -218,6 +251,15 @@ func resourceArmRedisCacheCreate(d *schema.ResourceData, meta interface{}) error
 
 	d.SetId(*read.ID)
 
+	// TODO: this _may_ need an `if premium` check
```

Comment:
Is this TODO for later or do we want to get it looked at now?

Reply:
good spot, I'd meant to remove that - I'll do that now..

Reply:
fixed.